webextensions/api/windows.json

```diff
@@ -1445,7 +1445,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤59"
+                  "version_added": "18"
```

Comment:
✅ https://github.com/chromium/chromium/commit/ad996343442c413edf82011c3e6b0e81b0af70ee in [18.0.1007.0](https://github.com/chromium/chromium/releases/tag/18.0.1007.0) before 18.0.1025.